frontend/src/static/js/v_summary.js

```diff
@@ -256,12 +257,12 @@ window.vSummary = {
 
         // filtering
         repo.users.forEach((user) => {
-          const toDisplay = this.filterSearch
+          const toDisplay = this.filterSearch.toLowerCase()
             .split(' ').filter(param => param)
             .map(param => user.searchPath.search(param) > -1)
             .reduce((curr, bool) => curr || bool, false);
 
-          if (!this.filterSearch || toDisplay) {
+          if (!this.filterSearch.toLowerCase() || toDisplay) {
```

Comment:
No need to convert lower case here, rather if `this.filterSearch` is `null` then we would get an error instead.

Reply:
Oh right, makes sense. 👍

Reply:
@yamidark requesting review